test/functional-portable/datastoresrp/noncloud/resources/testdata/datastoresrp-resources-sqldb-manual.bicep

```diff
@@ -100,5 +100,21 @@ resource sqlContainer 'Applications.Core/containers@2023-10-01-preview' = {
         }
       }
     }
+    runtimes: {
```

Comment:
are we adding this to resolve the sqlserver functional test issues?

Reply:
this was an attempt to try and limit the size of the sqlserver, I'll revert in a subsequent commit

Reply:
This is a pretty large amount of resources still (2 GB). Did this help?

Reply:
@rynowak per our offline discussion, do we want to go ahead and delete this test? If so, @superbeeny can do this as part of this PR which will also unblock it.

For context, @superbeeny, we added a functional test for every portable resource type initially because we had custom implementation for each of them. Since then the implementation has evolved and majority of the code has converged across all types, so it is covered extensively through other functional tests. SQL server specifically is also covered in Radius eshop sample.

Reply:
+1 to the consensus that this is ok to remove. It think that would mean deleting https://github.com/radius-project/radius/blob/main/test/functional-portable/datastoresrp/noncloud/resources/sql_test.go and the other files it references. 

There's actually more than one sql server test which is why I think we're hitting issues. That's 4gb minimum used by those tests.